pkg/cli/clisqlclient/conn.go

```diff
@@ -246,8 +242,8 @@ func (c *sqlConn) GetServerMetadata(
 ) (nodeID int32, version, clusterID string, err error) {
 	// Retrieve the node ID and server build info.
 	rows, err := c.Query(ctx, "SELECT * FROM crdb_internal.node_build_info")
-	if errors.Is(err, driver.ErrBadConn) {
```

Comment:
can you explain how this changed? does the driver close the connection instead now?

Reply:
this condition is just meant to detect when the connection to the server was lost. e.g. because the server was shut down, or the computer lost its network connection.